intent_brokering/README.md

```diff
@@ -210,7 +210,9 @@ This walkthrough is described in the [examples kv-app README](./examples/applica
 
 ## How to run the dog mode demo
```

Comment:
Perhaps this section should be removed?

Reply:
I wanted to leave it in for legacy reasons, in case people come here looking for dogmode, but I did updated the link and mentioned that it is hosted in the chariott-example-applications repository